cli/autocomplete.py

```diff
@@ -56,22 +56,15 @@
             '--quiet-error',
             '--identity',
             '--workspace-id',
-            '--static-routing-table',
-            '--routing-table',
-            '--dynamic-routing',
             '--extra-search-paths',
+            '--timeout-ctrl',
             '--uses',
             '--uses-with',
             '--uses-metas',
             '--uses-requests',
             '--py-modules',
             '--port-in',
-            '--port-out',
-            '--hosts-in-connect',
             '--host-in',
```

Comment:
is host in relevant? should it only be called host?

Reply:
yes I think it should be named host, port-in should also be port instead I think. But lets tidy this up in the end?